### PR TITLE
doc: Adding a contributing section to README.md

### PR DESCRIPTION
**Pull request by wristshot9987**
## Description
Added a contributions tab so visiting users who want to contribute to this GitHub can find the correct information.

- [x] I have read and understand the CONTRIBUTING.md document in this repository

## Type of change

- [x] This change requires a documentation update

## Checklist:

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Existing test suite passes locally with my changes
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,10 @@ Check out the [Payload website](https://payloadcms.com/docs/getting-started/what
 
 There are lots of good conversations and resources in our [GitHub Discussions board](https://github.com/payloadcms/payload/discussions). If you're struggling with something, chances are, someone's already solved what you're up against. Searching Discussions will often provide very helpful tips and tricks.
 
+##### Contributions
```
Comment (denolfe) [inline README.md:78]: I think this should go a bit higher. After `Documentation` as an H3 would be great.

File: README.md [hunk 2]
```diff
@@ -75,6 +75,10 @@ Check out the [Payload website](https://payloadcms.com/docs/getting-started/what
...
+
+If you want to add contributions to this repository, please follow the instructions in contributing.md.
```
Comment (denolfe) [inline README.md:80]: Let's make contributing.md a link to the doc `[contributing.md](./contributing.md)`

**Review by denolfe — CHANGES_REQUESTED** [review]
Just a few tweaks, then it looks good 👍

**Comment by wristshot9987** [comment]
I updated the README.md with your requests.

**Review by denolfe — APPROVED** [review]

**Comment by denolfe** [comment]
Thanks @wristshot9987!

**Pull request merged**